example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>11.0</string>
+  <string>12.0</string>
```

Comment:
This should be 13.0

Reply:
Need to look at what generates these (in any case not your fault :)

Reply:
For example app it was automatically set to 12. However, FoxFacts didn't like to be at 13. It was throwing some build errors.